src/test/java/com/cottonfactory/products/controllers/SlackControllerTest.java

```diff
@@ -0,0 +1,52 @@
+package com.cottonfactory.products.controllers;
+
+import com.cottonfactory.products.entities.Slack;
+import com.cottonfactory.products.services.SlackService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest
+public class SlackControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private SlackService slackService;
+// /api/products/slacks
+    @Test
```

Comment:
Remove this test since this does not add value.

Reply:
test removed